Resolve syncing conflicts from repo_sync_working_branch to live (#2621)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/provision-neo-hub.md

---

> title: Overview of the Microsoft New Employee Onboarding sites

Reviewer: ```suggestion
title: Overview of the Microsoft new employee onboarding sites
```

search.appverid:
- SPO160
- MET150

> description: "Overview of the Microsoft New Employee Onboarding sites"

Reviewer: ```suggestion
description: "Overview of the Microsoft new employee onboarding sites"
```

---

> # Overview of the Microsoft New Employee Onboarding sites

Reviewer: ```suggestion
# Overview of the Microsoft new employee onboarding sites
```

> There are three (3) new templates for New Employee Onboarding (NEO) to help organizations improve their onboarding process. The NEO sites are designed to deliver a flexible and consistent experience. The NEO sites can be used on their own, or together.

Reviewer: ```suggestion
There are three new templates for new employee onboarding (NEO) to help organizations improve their onboarding process. The NEO sites are designed to deliver a flexible and consistent experience. The NEO sites can be used on their own, or together.
```

The NEO site(s) helps organizations by: 

- Providing new employees a place to get started
- Connecting new employees to people and culture
- Helping stakeholders easily contribute to new employee onboarding
- Generating data and usage reports that help measure NEO content effectiveness

> ### Why invest in new employee onboarding sites?

Reviewer: ```suggestion
### Why invest in NEO sites?
```

> New employee onboarding (NEO) should be a strategic process that integrates new employees into an organization and its culture while providing the knowledge and tools needed to become fully contributing team members. 

Reviewer: ```suggestion
NEO should be a strategic process that integrates new employees into an organization and its culture while providing the knowledge and tools needed to become fully contributing team members. 
```

> NEO processes often fall short for both the new hires and the organization. Only 12% of employees strongly agree their organization does a great job of onboarding new employees. An engaging and well-organized NEO process can make all the difference in helping a new hire navigate through an exciting – but stressful -  career journey, and it can have major organizational benefits. 

Reviewer: ```suggestion
NEO processes often fall short for both the new hires and the organization. Only 12% of employees strongly agree their organization does a great job of onboarding new employees. An engaging and well-organized NEO process can make all the difference in helping a new hire navigate through an exciting–but stressful–career journey, and it can have major organizational benefits. 
```

- **Sample site pages:** To inspire and provide design templates for arranging your content, the NEO site includes sample inner site pages. Use these site pages as templates for your content. 
- **Easy provisioning:** Provision NEO sites from the SharePoint look book with just a few steps.
- **Mobile ready:** NEO sites can be easily accessed on a mobile device when you [get the SharePoint mobile app](https://www.microsoft.com/microsoft-365/sharepoint/mobile-app?wt.mc_id=AID686708_QSG_228389&ocid=AID686708_QSG_228389&rtc=1).

### NEO site template options: 
New employee onboarding involves multiple levels within an organization, including pre-onboarding, corporate onboarding, and departmental onboarding. Each onboarding level provides its own unique value, contributing to a comprehensive onboarding experience each new employee should experience. 

> Research has shown Pre-onboarding new hires, after they sign their acceptance letter but before they officially join the company, can lead to higher performance and better retention rates. To deliver a flexible and consistent onboarding experience, NEO sites consist of three different SharePoint site templates, that are designed to work alone or as one cohesive and familiar experience for new hires.

Reviewer: ```suggestion
Research has shown Pre-onboarding new hires after they sign their acceptance letter but before they officially join the company, can lead to higher performance and better retention rates. To deliver a flexible and consistent onboarding experience, NEO sites consist of three different SharePoint site templates that are designed to work alone or as one cohesive and familiar experience for new hires.
```

| Share the Pre-onboarding site with new hires as soon as the job offer has been accepted | Share the Corporate onboarding site with new hires on their first day| Managers and onboarding buddies should share their respective Departmental onboarding site |

<br>

1.	**New employee pre-onboarding site:** A site for new hires, who have yet to officially join the company, to learn more about the company they have joined and to get ready for their official start date. External guest access can be used for providing pre-start hires, with no corporate credentials, access to the Pre-onboarding site only. 
2.	**New employee corporate onboarding site:** A place for new hires to visit to get the information and connections they need to successfully onboard to the organization. 
3.	**New employee departmental onboarding site:** A place for new hires to visit to learn more about the department they are joining, its people, culture, and priorities. The Departmental onboarding site can be associated to an existing departmental hub.

> [!NOTE]

> > To deliver a flexible and consistent new hire onboarding experience the NEO sites consists of **three different SharePoint site templates**, that are designed to work alone or as one cohesive and familiar experience for new hires. Sites must be provisioned individually, and then can be configured to [create a hub](https://docs.microsoft.com/sharepoint/create-hub-site) or [add a site to an existing hub](https://support.microsoft.com/office/associate-a-sharepoint-site-with-a-hub-site-ae0009fd-af04-4d3d-917d-88edb43efc05#:~:text=Associate%20a%20SharePoint%20site%20with%20a%20hub%20site.,to%20your%20site.%20You%20can%20...%20More%20items).

Reviewer: ```suggestion
> To deliver a flexible and consistent new hire onboarding experience the NEO sites consist of **three different SharePoint site templates**, that are designed to work alone or as one cohesive and familiar experience for new hires. Sites must be provisioned individually, and then can be configured to [create a hub](https://docs.microsoft.com/sharepoint/create-hub-site) or [add a site to an existing hub](https://support.microsoft.com/office/associate-a-sharepoint-site-with-a-hub-site-ae0009fd-af04-4d3d-917d-88edb43efc05#:~:text=Associate%20a%20SharePoint%20site%20with%20a%20hub%20site.,to%20your%20site.%20You%20can%20...%20More%20items).
```

> > You must have site collection administrator permissions or higher to provision the New employee on baording sites.

Reviewer: ```suggestion
> You must have site collection administrator permissions or higher to provision the New employee onboarding sites.
```

## Step 1: Provision NEO sites

> The New Employee Onboarding (NEO) sites can be provisioned from the [SharePoint look book](https://lookbook.microsoft.com/). With the SharePoint look book, a **site collection administrator** can start the provisioning process with a few steps. Each site takes about 20 minutes to provision. Before starting the provisioning process, make sure you have met the prerequisites for provisioning.

Reviewer: ```suggestion
The NEO sites can be provisioned from the [SharePoint look book](https://lookbook.microsoft.com/). With the SharePoint look book, a **site collection administrator** can start the provisioning process with a few steps. Each site takes about 20 minutes to provision. Before starting the provisioning process, make sure you have met the prerequisites for provisioning.
```

### Prerequisites

> To successfully provision the NEO sites via the SharePoint look book, the person doing the provisioning must be a site collection admin of the tenant where ta NEO site(s) will be provisioned. If you have never provisioned a template from the Look book, [review overview guidance](https://docs.microsoft.com/sharepoint/add-sample-site). 

Reviewer: ```suggestion
To successfully provision the NEO sites via the SharePoint look book, the person doing the provisioning must be a site collection admin of the tenant where the NEO site(s) will be provisioned. If you have never provisioned a template from the look book, [review overview guidance](https://docs.microsoft.com/sharepoint/add-sample-site). 
```

### Provision the NEO sites

![Image of the NEO sites solution in the lookbook](media/neo-look-book.png)

1.	Go to the [NEO sites solution page](https://provisioning.sharepointpnp.com/details/3df8bd55-b872-4c9d-88e3-6b2f05344239).

  - Provision the New employee pre-onboarding site 
  - Provision the New employee corporate onboarding site 
  - Provision the New employee department onboarding site 

1.	Navigate to the site you want to provision and select **Add to your tenant**. If you are not signed into to your tenant, the SharePoint look book will ask for your site collection admin credentials.

1.	From the permissions requested dialog box, select **Consent on behalf of your organization** and then select **Accept**.

    > [!IMPORTANT]

>     > To deliverThe provisioning service requires these permissions to provision the site template. There is no overall impact on your tenant and these permissions are explicitly used for the purpose of the solution installation. You must accept these permissions to proceed with the installation.

Reviewer: ```suggestion
    > To deliver the provisioning service requires these permissions to provision the site template. There is no overall impact on your tenant and these permissions are explicitly used for the purpose of the solution installation. You must accept these permissions to proceed with the installation.
```

## Step 2: Customize the onboarding experience 

> The New Employee Onboarding (NEO) sites consist of three SharePoint site templates that can be customized to fit the needs of your users and organization. Many of the core pages are already built and pre-populated with content. Review content on sites and pages, then plan on customizing content, images, branding, web parts, and pages.   

Reviewer: ```suggestion
The new employee onboarding (NEO) sites consist of three SharePoint site templates that can be customized to fit the needs of your users and organization. Many of the core pages are already built and pre-populated with content. Review content on sites and pages, then plan on customizing content, images, branding, web parts, and pages.   
```

1.	Open your web browser and navigate to [office.com](https://www.office.com) or your organization’s sign-in location.

2.	**Sign in** with your username and password.

3.	Navigate to the location of the site using the URL supplied by your site collection admin, or select SharePoint from the Microsoft 365 home page, and then select the site.

### Explore and review pre-populated content

>  To deliver a flexible and consistent new hire onboarding experience the NEO sites consists of three different SharePoint site templates, that are designed to work alone or as one cohesive and familiar experience for new hires. Check with your site collection administrator to confirm which NEO sites were provisioned and are available for customizations. 

Reviewer: ```suggestion
 To deliver a flexible and consistent new hire onboarding experience the NEO sites consist of three different SharePoint site templates that are designed to work alone or as one cohesive and familiar experience for new hires. Check with your site collection administrator to confirm which NEO sites were provisioned and are available for customizations. 
```

- **Welcome** – Give new hires a warm welcome and place to start understanding onboarding tasks and how to prepare for their first day. This is a good opportunity to include a video message from leadership. Use the [YouTube web part](https://support.microsoft.com/office/use-the-youtube-web-part-c14fa2c1-71dc-4e52-94b6-b4876742382f), or [Embed web part](https://support.microsoft.com/office/add-content-to-your-page-using-the-embed-web-part-721f3b2f-437f-45ef-ac4e-df29dba74de8) to display the video.
- **Contoso 101** – Provide high-level information about the organization that engages and excites. Share more about leadership and values in Our leadership team and Our values pages.
- **Prepare for your first day** – Ensure new hires feel prepared and supported on their first day by providing details on what to bring and where to go. 
- **Help & Support** – Highlight where to go for support and customize questions and answers for the FAQ page.

#### New employee corporate onboarding site:

![Image the corporate onboarding site](media/neo-corp-onboarding.png)

> The corporate onboarding site is the landing place for the new employee onboarding experience and is designed to provide a high-level view of organizational goals, leadership, team structure, and resources. In this phase, users are looking for guidance, support, and clarity. Use this site to outline onboarding details and expectations during the first months of onboarding. Ensure users have access to support channels like [Yammer](https://support.microsoft.com/en-us/office/use-a-yammer-web-part-in-sharepoint-online-a53cfa0c-3d09-42c8-a286-1038a81c59da), write FAQs relevant to your organization, and customize the [onboarding checklist](https://support.microsoft.com/office/introduction-to-lists-0a1c3ace-def0-44af-b225-cfa8d92c52d7) to include the activities you want all new hires to do in their first 30, 60, 90 days.

Reviewer: ```suggestion
The corporate onboarding site is the landing place for the new employee onboarding experience and is designed to provide a high-level view of organizational goals, leadership, team structure, and resources. In this phase, users are looking for guidance, support, and clarity. Use this site to outline onboarding details and expectations during the first months of onboarding. Ensure users have access to support channels like [Yammer](https://support.microsoft.com/en-us/office/use-a-yammer-web-part-in-sharepoint-online-a53cfa0c-3d09-42c8-a286-1038a81c59da), write FAQs relevant to your organization, and customize the [onboarding checklist](https://support.microsoft.com/office/introduction-to-lists-0a1c3ace-def0-44af-b225-cfa8d92c52d7) to include the activities you want all new hires to do in their first 30, 60, and 90 days.
```

 
##### Pre-populated site contents:

> - **Home page** – Provide a high-level view of significant concepts that will be relevant to new users. This page is a great location to help new employees build their network and learn from more experienced and knowledgeable employees with [Microsoft Teams live events](https://docs.microsoft.com/microsoftteams/teams-live-events/what-are-teams-live-events#:~:text=Microsoft%20365%20live%20events%20bring%20live%20video%20streaming,community%20resides%2C%20using%20Microsoft%20Stream%2C%20Teams%2C%20or%20Yammer.)

Reviewer: ```suggestion
- **Home page** – Provide a high-level view of significant concepts that will be relevant to new users. This page is a great location to help new employees build their network and learn from more experienced and knowledgeable employees with [Microsoft Teams live events](https://docs.microsoft.com/microsoftteams/teams-live-events/what-are-teams-live-events#:~:text=Microsoft%20365%20live%20events%20bring%20live%20video%20streaming,community%20resides%2C%20using%20Microsoft%20Stream%2C%20Teams%2C%20or%20Yammer.).
```

> - **Start here** – Specify what new hires should do in their first 30, 60, and 90 days of onboarding by creating an onboarding process in on the Start your journey here page. The new hire checklist found in this section comes pre-populated with a set of generic onboarding activities. Customize list content to meet your needs. [Learn more about working with SharePoint lists](https://support.microsoft.com/office/introduction-to-lists-0a1c3ace-def0-44af-b225-cfa8d92c52d7).

Reviewer: ```suggestion
- **Start here** – Specify what new hires should do in their first 30, 60, and 90 days of onboarding by creating an onboarding process on the Start your journey here page. The new hire checklist found in this section comes pre-populated with a set of generic onboarding activities. Customize list content to meet your needs. [Learn more about working with SharePoint lists](https://support.microsoft.com/office/introduction-to-lists-0a1c3ace-def0-44af-b225-cfa8d92c52d7).
```

> - **Who we are** – Introduce users to more detail about the organization in the Our story, Our leadership, Our teams pages. Customize these pages and the Office locations page for your organization. Or, link to an existing leadership page instead.

Reviewer: ```suggestion
- **Who we are** – Introduce users to more detail about the organization in the Our story, Our leadership, and Our teams pages. Customize these pages and the Office locations page for your organization. Or, link to an existing leadership page instead.
```

- Train site owners and authors. Make sure all site owners and authors have appropriate training and access to [maintain the site](https://support.microsoft.com/office/manage-your-sharepoint-communication-site-21761aac-f7f7-4499-b0ca-cf283477c32f).
- Update content in web parts. Keep web parts like the People web part updated to ensure you are using the full value of the NEO sites.
- Review previously established metrics after the launch. Use insights from [site analytics](https://support.microsoft.com/office/view-usage-data-for-your-sharepoint-site-2fa8ddc2-c4b3-4268-8d26-a772dc55779e) to promote content on the home page, update navigation, or rewrite content for clarity.
- Establish a schedule to audit content. Plan when site owners should audit content in advance to make sure your sites are always up to date.
- Periodically review your site settings. Make [changes to the settings](https://support.microsoft.com/office/manage-your-sharepoint-site-settings-8376034d-d0c7-446e-9178-6ab51c58df42?ui=en-us&rs=en-us&ad=us), site information, and permissions for the site as needed.

## NEO site FAQs

> **Question**: What are the requirements for installing the New Employee Onboarding (NEO) sites into my tenant environment?

Reviewer: ```suggestion
**Question**: What are the requirements for installing the new employee onboarding (NEO) sites into my tenant environment?
```

<br>

**Answer**:
- SharePoint and communication sites enabled.

> - The individual that will be provisioning CLO365 must be the site collection administrator of the target tenant for install.

Reviewer: ```suggestion
- The individual that will be provisioning CLO365 must be the site collection administrator of the target tenant for installation.
```

**Question**: Where can I ask questions or report an issue with the SharePoint provisioning service? 
<br>

> **Answer**:Post your questions and comments about the provisioning of the NEO sites to the SharePoint [provisioning service forum](https://github.com/SharePoint/sp-provisioning-service/issues).

Reviewer: ```suggestion
**Answer**: Post your questions and comments about the provisioning of the NEO sites to the SharePoint [provisioning service forum](https://github.com/SharePoint/sp-provisioning-service/issues).
```

**Question**: Where can I share feedback for ideas on how to improve the SharePoint site experience? 
<br>

> **Answer**:Post your ideas and comments about SharePoint sites in the [Site and Collaboration User Voice Forum](https://sharepoint.uservoice.com/forums/329214-sites-and-collaboration)

Reviewer: ```suggestion
**Answer**: Post your ideas and comments about SharePoint sites in the [Site and Collaboration User Voice Forum](https://sharepoint.uservoice.com/forums/329214-sites-and-collaboration).
```

**Question**: Where can I ask questions or report an issue with the SharePoint provisioning service? 
<br>

> **Answer**:Post your questions and comments about the provisioning of the NEO sites to the SharePoint [provisioning service forum](https://sharepoint.uservoice.com/forums/329214-sites-and-collaboration?category_id=144408).

Reviewer: ```suggestion
**Answer**: Post your questions and comments about the provisioning of the NEO sites to the SharePoint [provisioning service forum](https://sharepoint.uservoice.com/forums/329214-sites-and-collaboration?category_id=144408).
```
